staging/src/k8s.io/component-base/logs/options.go

```diff
@@ -90,4 +97,9 @@ func (o *Options) apply() {
 		panic(fmt.Errorf("internal error while setting klog vmodule: %v", err))
 	}
 	klog.StartFlushDaemon(o.Config.FlushFrequency)
+	contextualLoggingEnabled := FeatureGates[ContextualLogging].Default
+	if featureGate != nil {
+		contextualLoggingEnabled = featureGate.Enabled(ContextualLogging)
+	}
```

Comment:
I like this. can accomplish the same thing without exposing a package-level `FeatureGates` by referencing a private `contextualLoggingDefault bool` here and in logs.AddFeatureGates

Reply:
Done.